build-system/tasks/e2e/index.js

```diff
@@ -48,33 +45,12 @@ function buildRuntime_() {
   execOrDie(`gulp dist --fortesting --config ${argv.config}`);
 }
 
-function launchWebServer_() {
-  log('Launching webserver at', cyan(`http://${HOST}:${PORT}`) + '...');
-  webServerProcess_ = execScriptAsync(
-    `gulp serve --compiled --host ${HOST} --port ${PORT}`,
-    {stdio: 'ignore'}
```

Comment:
I believe `{stdio: 'ignore'}` was needed to prevent Travis from hanging due to a node process still running. Same applies to the visual-diff task.

Reply:
If it helps, I tried running `gulp visual-diff` twice and both times it appeared to be hanging after tests completed:

<img width="568" alt="Screen Shot 2019-11-18 at 3 41 10 PM" src="https://user-images.githubusercontent.com/10456171/69092528-8f6e0100-0a1a-11ea-8b6f-be21ac4b514e.png">

Reply:
Both useful comments. I'm seeing this too. Lemme see if I can find a fix.

Reply:
Turns out that `visual-diff` and `e2e` were triggering the lazy build workflow, causing the server to not fully shutdown after all tests were run. I've refactored the server code to prevent this, and initial testing suggests the problem is fixed.